torchaudio/prototype/pipelines/hifigan_pipeline.py

```diff
@@ -0,0 +1,155 @@
+from dataclasses import dataclass
+from typing import Any, Dict
+
+from torch.nn import Module
+from torchaudio._internal import load_state_dict_from_url
+
+from torchaudio.prototype.models.hifi_gan import hifigan_generator, HiFiGANGenerator, HiFiGANMelSpectrogram
+
+
+@dataclass
+class HiFiGANVocoderBundle:
+    """Data class that bundles associated information to use pretrained
+    :py:class:`~torchaudio.prototype.models.HiFiGANGenerator`.
+
+    This class provides interfaces for instantiating the pretrained model along with
+    the information necessary to retrieve pretrained weights and additional data
+    to be used with the model.
+
+    Torchaudio library instantiates objects of this class, each of which represents
+    a different pretrained model. Client code should access pretrained models via these
+    instances.
+
+    This bundle can convert mel spectrorgam to waveforms and vice versa. A typical use case would be a flow like
+    `text -> mel spectrogram -> waveform`, where one can use an external component, e.g. Tactron2,
+    to generate mel spectrogram from text. Please see below for the code example.
+
+    .. note::
+        Although this works with the existing Tactron2 bundles, for the best results one needs to retrain Tactron2
+        using the same data preprocessing pipeline which was used for training HiFiGAN. In particular, the original
+        HiFiGAN implemntation uses a custom method of generating mel spectrograms from waveforms, different from
+        :py:class:`torchaudio.transforms.MelSpectrogram`. We reimplemented this transform as
+        :py:meth:`get_mel_transform`.
```

Comment:
This note is specific to `HIFIGAN_VOCODER_V3_LJSPEECH` instance, so it is more appropriate to move there.

Reply:
Moved it to the docstring of `HIFIGAN_VOCODER_V3_LJSPEECH`